src/Entities/Applicant.php

```diff
@@ -12,18 +12,24 @@ class Applicant
     private string $application_date;
     private ?Application $application;
 
+
     public function __construct(
-        int $id,
-        string $name,
+        int          $id,
+        string       $name,
         EmailAddress $email,
-        string $application_date,
+        string       $application_date,
+
         ?Application $application = null
-    ) {
+
```

Comment:
Don't format like this unless CircleCI tells you to - inconsistent with the rest of the app.